cmd/klotho/dependencies.go

```diff
@@ -4,12 +4,13 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/klothoplatform/klotho/pkg/logging"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"runtime"
 
+	"github.com/klothoplatform/klotho/pkg/logging"
```

Comment:
weird spacing here

Reply:
What's weird about the spacing? Imports are automatically formatted by the go LSP (which uses [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) style).

Reply:
it looks like there's an extra blank space on line 13?

Reply:
That's because it separates stdlib, internal, and external packages